assets/arc-0052/contextual.api.crypto.ts

```diff
@@ -0,0 +1,292 @@
+import {
+    crypto_core_ed25519_scalar_add,
+    crypto_core_ed25519_scalar_mul,
+    crypto_core_ed25519_scalar_reduce,
+    crypto_hash_sha512,
+    crypto_scalarmult_ed25519_base_noclamp,
+    crypto_sign_verify_detached,
+    ready,
+    crypto_sign_ed25519_pk_to_curve25519,
+    crypto_scalarmult
+} from 'libsodium-wrappers-sumo';
+import * as msgpack from "algo-msgpack-with-bigint"
+import Ajv from "ajv"
+import { deriveChildNodePrivate, fromSeed } from './bip32-ed25519';
+
+
+/**
+ * 
+ */
+export enum KeyContext {
+    Address = 0,
+    Identity = 1,
+    Cardano = 2,
```

Comment:
Do we really need Cardano as a key context

Reply:
Ah we don't. This was just to try to use some of their test vectors to match implementations. I can remove it